README.md

```diff
@@ -171,6 +171,52 @@ system). If your container gets messed up you can blow it away by stopping it an
 
 
+## Python support (WIP)
+
+There is a **WIP** Python API which is available on the master branch. You will need to [build the SDK](#building-the-sdk) 
+if you wish to experiment with it. Please [let us know](#community) how you got on.
+
+The [enclave Java API](https://docs.conclave.net/api/-conclave%20-core/com.r3.conclave.enclave/-enclave/index.html) 
+has been ported to the following global functions:
+
+* `on_enclave_startup()` - equivalent to [`onStartup`](https://docs.conclave.net/api/-conclave%20-core/com.r3.conclave.enclave/-enclave/on-startup.html)
+* `on_enclave_shutdown()` - equivalent to [`onShutdown`](https://docs.conclave.net/api/-conclave%20-core/com.r3.conclave.enclave/-enclave/on-shutdown.html)
+* `receive_from_untrusted_host(bytes)` - equivalent to [`receiveFromUntrustedHost`](https://docs.conclave.net/api/-conclave%20-core/com.r3.conclave.enclave/-enclave/receive-from-untrusted-host.html).
+  The Java byte array is converted to Python [`bytes`](https://docs.python.org/3/library/stdtypes.html#bytes-objects).
+  If there’s no return value then it is treated as null, otherwise the return value is expected to be `bytes`.
+* `receive_enclave_mail(mail)` - equivalent to [`receiveMail`](https://docs.conclave.net/api/-conclave%20-core/com.r3.conclave.enclave/-enclave/receive-mail.html).
+  The Java [`EnclaveMail`](https://docs.conclave.net/api/-conclave%20-core/com.r3.conclave.mail/-enclave-mail/index.html)
+  object is converted to a simpler Python equivalent which is just a class holding the body, envelope and 
+  authenticated sender. The topic and sequence number are ignored for now. The authenticated sender is represented 
+  by its encoded binary form in `bytes`. This function is also much simpler than `receiveMail` in that the response can 
+  simply be returned from the function and it will encrypted as a response mail back to the sender key and posted.
+
+These functions need to be defined in a single Python file and are all optional. Not defining them is equivalent to 
+not overriding the equivalent method from `Enclave`. The Python script must exist in the enclave Gradle module under 
+`src/main/python`. Only one Python script is supported at this time. Otherwise, everything else is the same as a 
+Java or Kotlin project. The Python enclave module needs to be part of a Gradle multi-module project with the 
+host module taking a dependency to the enclave module. You can take the [hello world code](https://github.com/R3Conclave/conclave-tutorials/tree/master/hello-world)
+and modify the `enclave` module accordingly. 
+
+The Python script only has access to an `enclave_sign(data)` global function, which allows the given data `bytes` to be 
+signed by the enclave's private signing key. This is equivalent to [`signer()`](https://docs.conclave.net/api/-conclave%20-core/com.r3.conclave.enclave/-enclave/signer.html)
+in the Java API.
```

Comment:
Converted this into bullet points. Also, this section is only in the README whilst it's WIP. Once we're more happy with it, the doc for it will move to the docsite. We can probably do this if want to announce the feature as beta in the next release.